.github/workflows/check.yml

```diff
@@ -14,6 +14,14 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Install packages (Linux)
+        if: runner.os == 'Linux'
+        uses: awalsh128/cache-apt-pkgs-action@v1.4.2
+        with:
+          packages: libxcb-render0-dev libxcb-shape0-dev libxcb-xfixes0-dev libxkbcommon-dev libssl-dev libgtk-3-dev # libgtk-3-dev is used by rfd
+          version: 1.0
+          execute_install_scripts: true
+
```

Comment:
Why is this needed?

Reply:
https://github.com/lucasmerlin/hello_egui/actions/runs/11125313206/job/30912734374

this happens otherwise

Reply:
btw still happens - still trying to fix

Reply:
this came up as i removed the exclude from clippy